addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -36,7 +36,7 @@ class AccountMove(models.Model):
             ('08', '08 Penyerahan yang PPN-nya Dibebaskan (Impor Barang Tertentu)'),
             ('09', '09 Penyerahan Aktiva ( Pasal 16D UU PPN )'),
         ], string='Kode Transaksi', help='Dua digit pertama nomor pajak',
-        readonly=True, states={'draft': [('readonly', False)]}, copy=False)
+        readonly=True, states={'draft': [('readonly', False)]}, copy=False, compute="_compute_kode_transaksi")
```

Comment:
Would be good to see if this thing changes per invoice or is always the same for that partner, but you need a store True here to not violate stable policy.

Reply:
With the store, once the invoice has been posted (which is the case in PoS), the kode_transaksi has the value of the partner_id's kode at the time of invoicing. Subsequent changes of the kode in the customer contact doesn't affect already created invoices. (which I presume is the desired behaviour)